### DISCUSSION TITLE
Welcome to Finnix Discussions!

### DISCUSSION THREAD

**Discussion by rfinnie** [finnix/finnix#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hi. I would like to see the ixsi perl tool included in Finnix. Maybe it's already there, I have not really checked yet :)
Best regards

**Comment by hjadb** [comment]
How can I add our image to your image list.  https://mirrors.finnix.org/
Or you can add it for me. That's even better

Please refer to the following information.
1. Country: CN
2. Site Name: Alibaba Cloud
3. Mirror Name: mirrors.aliyun.com
4. Categories: http(http://mirrors.aliyun.com/finnix/), https (https://mirrors.aliyun.com/finnix/).
5. Bandwidth: 10000
6. Internet2: No
7. Email: ali-yum@alibaba-inc.com

Thank you

**Comment by djringjr** [comment]
Hello,

Is thee any way I can take a look at the excellent script "wifi-connect"?  Also can it be used with credit being given to the author?  Can it be modifieid?

Thanks,

David Ring 
djringjr
